.gitlab-ci.yml

```diff
@@ -242,6 +242,11 @@ before_script:
     - make -f Makefile.ci -j "$NJOBS" "${CI_JOB_NAME#*:}"
     - echo 'end:coq.test'
     - set +e
+  artifacts:
+    name: "$CI_JOB_NAME"
+    paths:
+      - _build_ci
+    when: always
```

Comment:
Maybe we should set expire_in instead of relying on the default (currently 30 days https://gitlab.com/help/user/gitlab_com/index.html#gitlab-cicd)